pkg/controllers/disruption/controller.go

```diff
@@ -61,23 +56,11 @@ const pollingPeriod = 10 * time.Second
 
 var errCandidateDeleting = fmt.Errorf("candidate is deleting")
 
-// waitRetryOptions are the retry options used when waiting on a NodeClaim to become ready or to be deleted
-// readiness can take some time as the node needs to come up, have any daemonset extended resoruce plugins register, etc.
-// deletion can take some time in the case of restrictive PDBs that throttle the rate at which the node is drained
-func waitRetryOptions(ctx context.Context) []retry.Option {
-	return []retry.Option{
-		retry.Context(ctx),
-		retry.Delay(2 * time.Second),
-		retry.LastErrorOnly(true),
-		retry.Attempts(60),
-		retry.MaxDelay(10 * time.Second), // 22 + (60-5)*10 =~ 9.5 minutes in total
-	}
-}
-
 func NewController(clk clock.Clock, kubeClient client.Client, provisioner *provisioning.Provisioner,
-	cp cloudprovider.CloudProvider, recorder events.Recorder, cluster *state.Cluster) *Controller {
-
+	cp cloudprovider.CloudProvider, recorder events.Recorder, cluster *state.Cluster, queue *orchestration.Queue) *Controller {
+	c := makeConsolidation(clk, cluster, kubeClient, provisioner, cp, recorder, queue)
```

Comment:
This is now using the same `consolidation` struct vs. using different ones like the previous iteration. Can we just do a sanity check to ensure that this doesn't cause weird behavior with stateful information

Reply:
Sure, sanity checking here, let me see if you think anything to the contrary.
- queue: uses a pointer as expected, since each consolidation should refer to the same queue.
- clock: uses a separate clock for each. Our testing uses a pointer here to control time across the controller, but each clock is separate, incrementing all together. Could see an argument to make this a pointer, but this is the same behavior as before. 
- cluster: uses pointer as expected, no change
- kubeClient: uses interface, no needed shared information is in the client.
- provisioners: uses pointer, no change
- cloudProvider: interface, no needed shared information
- recorder: interface, no needed shared information
- lastConsolidationState: keeps track of the last time each consolidation did nothing, and compares it to the cluster state's timestamp, which is important to keep separate (which this code doesn't change).